processmanager/processinfo.go

```diff
@@ -121,7 +121,7 @@ func (pm *ProcessManager) updatePidInformation(pr process.Process, m *Mapping) (
 			meta:             pr.GetProcessMeta(process.MetaConfig{IncludeEnvVars: pm.includeEnvVars}),
 			mappings:         make(map[libpf.Address]*Mapping),
 			mappingsByFileID: make(map[host.FileID]map[libpf.Address]*Mapping),
-			tsdInfo:          nil,
+			libcInfo:          nil,
```

Comment:
nit: is the `nil` indent not on same level as others? strange that this wasn't flagged by CI as style issue